addOns/reports/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Added
 - "XML Plus" report format for XML with requests and responses
 - Tags to "JSON Plus" report.
+- SARIF reporting
```

Comment:
This should be moved to the `Unreleased` section.

Reply:
Hmm...
I think there is also a bug inside the markdown indention of the existing change log:


> # Changelog
> All notable changes to this add-on will be documented in this file.
> 
> The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
> 
> ## Unreleased
> # Changed
> - The XML and JSON reports now include programName metadata elements (Issue 6640).
> 
> ## [0.16.0] - 2022-10-27
> ### Added
> 
According to https://keepachangelog.com/en/1.0.0/ I will change it to
```
## Unreleased
# Changed
```
to 

```
## Unreleased
### Added
- SARIF reporting

### Changed
```
And also add a `###Added`below `##Unreleased`

Reply:
Yup the headings are currently wrong